Make.defs

```diff
@@ -2,18 +2,29 @@ CC = gcc
 INSTALL = install -p
 MKDIR = mkdir
 
+# Flag to enable -Werror on build.
+WERROR =
+
 DBGFLAG = #-ggdb
 ifndef CFLAGS
 CFLAGS = -O2
 endif
-CFLAGS += $(DBGFLAG) -I../include -Wall
+CFLAGS += $(DBGFLAG) -I../include
+CFLAGS += -W -Wall -Wextra -Wpedantic
+ifneq ($(WERROR),)
+CFLAGS += -Werror
+endif
 #CFLAGS += -DFAIL_RECORD
 LDFLAGS += $(DBGFLAG) -L../libs
 
+PREFIX = /usr
 SBINDIR = /sbin
 SLIBDIR = /lib
```

Comment:
I don't mind adding `PREFIX`, but there are at least two locations (`SBINDIR` and `SLIBDIR`) that are currently do not start with prefix.

Reply:
Well, I didn't change them for the sake of portability to distributions, that still have or support a separate partition for `/usr`.